start/blobstore.md

```diff
@@ -153,7 +153,7 @@ context.close();
 {% endhighlight %}
 
 There are many options available for creating a `Context`.  Please see the
-[ContextBuilder](http://javadocs.jclouds.cloudbees.net/org/jclouds/ContextBuilder.html) Javadocs for
+[ContextBuilder](https://jclouds.apache.org/reference/javadoc/1.9.x/org/jclouds/ContextBuilder.html) Javadocs for
```

Comment:
Thanks for spotting that, @mss! The actual new master link is http://jclouds-javadocs.elasticbeanstalk.com/org/jclouds/ContextBuilder.html - could you kindly use that instead?